hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/AbstractFilterRel.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hazelcast.sql.impl.calcite.opt;
+package com.hazelcast.jet.sql.impl.opt;
```

Comment:
I think if you simply extend `Filter` for `com.hazelcast.jet.sql.impl.opt.physical.FilterPhysicalRel`, you could remove this class.